jobserver/models.py

```diff
@@ -196,6 +197,10 @@ class JobRequest(models.Model):
 
     created_at = models.DateTimeField(auto_now_add=True)
 
+    @property
+    def completed_at(self):
+        return first(reversed(self.ordered_jobs)).completed_at
```

Comment:
I think this could do with a check for `is_complete` first, to be sure no leaves are still running?

Reply:
Fixed and rebased into 34fcc16.  I also caught that there will be a window when `JobRequest` has no `Job`s so checking that too.